testutil/singlevm/setup.sh

```diff
@@ -79,6 +82,33 @@ done
 
 set -o nounset
 
+echo "Generating configuration file $conf_file"
+(
+cat <<-EOF
+configure:
+  scheduler:
+    storage_uri: /etc/ciao/configuration.yaml
+  storage:
+    ceph_id: ciao
+  controller:
+    compute_ca: /etc/pki/ciao/controller_cert.pem
+    compute_cert: /etc/pki/ciao/controller_key.pem
+    identity_user: csr
+    identity_password: hello
+  image_service:
+    type: glance
+    url: https://${ciao_host}
+  launcher:
+    compute_net: [198.51.100.1/24]
```

Comment:
@bwarden we should be able to use ${ciao_subnet} as it is set to ciao_subnet=198.51.100.1/24